Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift

```diff
@@ -3,18 +3,25 @@ import Library
 import Prelude
 import UIKit
 
+protocol PledgeCreditCardViewDelegate: AnyObject {
+  func pledgeCreditCardViewSelected(
+    _ pledgeCreditCardView: PledgeCreditCardView,
+    paymentSourceId: String
+  )
+}
+
 final class PledgeCreditCardView: UIView {
   // MARK: - Properties
 
-  private let viewModel: CreditCardCellViewModelType = CreditCardCellViewModel()
-
   private let adaptableStackView: UIStackView = { UIStackView(frame: .zero) }()
+  weak var delegate: PledgeCreditCardViewDelegate?
   private let expirationDateLabel: UILabel = { UILabel(frame: .zero) }()
   private let imageView: UIImageView = { UIImageView(frame: .zero) }()
   private let labelsStackView: UIStackView = { UIStackView(frame: .zero) }()
   private let lastFourLabel: UILabel = { UILabel(frame: .zero) }()
   private let rootStackView: UIStackView = { UIStackView(frame: .zero) }()
   private let selectButton: UIButton = { UIButton(type: .custom) }()
+  private let viewModel: CreditCardCellViewModelType = CreditCardCellViewModel()
```

Comment:
I mentioned in #835 that we should probably rename this 😬

Reply:
Yeah I agree - I really don't think we should have used the same view model as the one in the `Settings` screen at all though :\

Reply:
Oh I had no idea it was being used in both places.